src/Microsoft.VisualBasic/tests/StringsTests.cs

```diff
@@ -174,12 +174,175 @@ public void Filter_Objects_WhenObjectCannotBeConvertedToString_ThrowsArgumentOut
         [Theory]
         [InlineData(new object[] { 42 }, "42", new string[] { "42" }, new string[] { })]
         [InlineData(new object[] { true }, "True", new string[] { "True" }, new string[] { })]
-        public void Objects(object[] source, string match, string[] includeExpected, string[] excludeExpected)
+        public void Filter_Objects(object[] source, string match, string[] includeExpected, string[] excludeExpected)
         {
             Assert.Equal(includeExpected, Strings.Filter(source, match, Include: true));
             Assert.Equal(excludeExpected, Strings.Filter(source, match, Include: false));
         }
 
+        [Theory]
+        [InlineData(null, null, 0)]
+        [InlineData(null, "", 0)]
+        [InlineData("", null, 0)]
+        [InlineData("", "", 0)]
+        [InlineData(null, "a", 0)]
```

Comment:
These 5 values are common in some tests. we can move them to a function and supply them to the test using MemberData(nameof(function))
https://github.com/dotnet/corefx/blob/master/src/Common/tests/Tests/System/IO/PathInternal.Tests.cs#L193

Reply:
Its generally nice to use memberData attribute when a lot of inline Data elements are present. I will recommend it for ```InStr_FromBegin```, ```InStr_FromWithin```, ```InStrRev_FromEnd``` and ```InStrRev_FromWithin```